components/statuslight/stories/statuslight.stories.js

```diff
@@ -71,22 +81,193 @@ export default {
 			type: "migrated",
 		},
 	},
+	decorators: [
+		(Story, context) => {
+			if (!window.isChromatic()) return Story(context);
+			return html`
+				<style>
+					.spectrum-Detail { display: inline-block; }
+					.spectrum-Typography > div {
+						border: 1px solid var(--spectrum-gray-200);
+						border-radius: 4px;
+						padding: 0 1em 1em;
+						/* Why seafoam? Because it separates it from the component styles. */
+						--mod-detail-font-color: var(--spectrum-seafoam-900);
+					}
+				</style>
+				<div
+					style=${styleMap({
+						display: "flex",
+						flexDirection: "column",
+						alignItems: "flex-start",
+						gap: "1rem",
+						"--mod-detail-margin-end": ".3rem",
+					})}
+				>
+					${Story(context)}
+				</div>
+			`;
+		}
+	],
 };
 
-export const Default = (args) => html`
-	<div>
-		${Template({
-			...args
-		})}
+const Sizes = (args) =>
+	html` ${["s", "m", "l", "xl"].map((size) => {
+		return html`
+			<div>
+				${Typography({
+					semantics: "detail",
+					size: "s",
+					content: [
+						{
+							s: "Small",
+							m: "Medium",
+							l: "Large",
+							xl: "Extra-large",
+						}[size],
+					],
+				})}
+				${Template({
+					...args,
+					size
+				})}
+				<!-- Save truncation for VRTs -->
```

Comment:
Got it. I kept it in there since it was there before, but I'm all for deleting it.

https://github.com/adobe/spectrum-css/pull/2818/commits/ad6925c848517c8e94aec3b53bfcb546ddf37125